src/main/java/com/yelp/nrtsearch/server/config/LuceneServerConfiguration.java

```diff
@@ -45,6 +45,7 @@ public class LuceneServerConfiguration {
   public static final Path DEFAULT_INDEX_DIR =
       Paths.get(DEFAULT_USER_DIR.toString(), "default_index");
   private static final String DEFAULT_BUCKET_NAME = "DEFAULT_ARCHIVE_BUCKET";
+  static final int DEFAULT_MAX_S3_CLIENT_RETRIES = -1;
```

Comment:
Instead of -1 should we have a default which does retry some sane number of times? I feel this would be good to have by default and also seems safe enough

Reply:
The thought was to make this a noop unless set, so it was a very safe change. The sdk default is 3, we could maybe update this after we've done some testing of alternate values.